build/webpack.base.conf.js

```diff
@@ -0,0 +1,68 @@
+const path = require('path');
+const webpack = require('webpack');
+const moment = require('moment');
+
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const ReplaceInFileWebpackPlugin = require('replace-in-file-webpack-plugin');
+
+function resolve(dir) {
```

Comment:
hey, checkout another way to "resolve": https://github.com/CorpGlory/express-webpack-example/blob/master/build/webpack.base.conf.js#L13